services/core-api/app/api/mines/project_summary/resources/project_summary.py

```diff
@@ -105,6 +105,7 @@ def get(self, mine_guid, project_summary_guid):
             'mine_guid': 'The GUID of the mine the Project Description belongs to.',
             'project_summary_guid': 'The GUID of the Project Description to update.'
         })
+    @requires_any_of([MINE_ADMIN, MINESPACE_PROPONENT])
     @api.marshal_with(PROJECT_SUMMARY_MODEL, code=200)
```

Comment:
'MINE_ADMIN' is fine for now, but let's remember to update it when we implement the editing on the core side with the new role